src/core/stack_switching/esbuild.config.mjs

```diff
@@ -4,10 +4,38 @@
  */
 
 import { build } from "esbuild";
+import { readFileSync } from "node:fs";
+import loadWabt from "../../js/node_modules/wabt/index.js";
 import { dirname, join } from "node:path";
 
 const __dirname = dirname(new URL(import.meta.url).pathname);
 
+const { parseWat } = await loadWabt();
+
+/**
+ * An esbuild plugin to handle wat imports. It uses the wasm binary toolkit to
```

Comment:
There was https://github.com/mitschabaude/esbuild-plugin-wat but it doesn't look maintained.

Reply:
In any case, for 17 lines of code I'd rather spin our own than bring in a dependency. Of course we're depending on the wasm binary toolkit here but I think that has a good chance of being maintained.